website/src/pages/api/subscriptions/[...path].ts

```diff
@@ -0,0 +1,81 @@
+import { getSession } from 'auth-astro/server';
+import { getDashboardsConfig } from '../../../config.ts';
+
+const API_PATHNAME_LENGTH = '/api'.length;
+
+export async function ALL({ request }: { request: Request }) {
+    const session = await getSession(request);
+
+    if (!session || !session.user?.id) {
+        return getUnauthorizedResponse(request.url);
+    }
+
+    const backendUrl = getBackendUrl(request, session.user.id);
+
+    try {
+        const response = await fetch(backendUrl, {
+            method: request.method,
+            body: request.body,
+        });
```

Comment:
POST requests don't work yet. I think we can simply do this:
```suggestion
        const response = await fetch(backendUrl, request);
```

Reply:
POST works now